docs/source/examplecleanup.rst

```diff
@@ -6,55 +6,94 @@ End-to-End Example
-      # this is the number of individual bad password hits tartufo has found:
-      ls ${TARTUFOSCANPATH} | wc -l
+      git clone --mirror git@${GITHUBADDRESS}:${GITHUBPROJECT}/${GITHUBREPO}
 
 
-#. Take results and create a "bad password" file. 
+#. Use ``tartufo`` to scan your repository and find any secrets in its history!
 
-   This file will be used by ``BFG`` and replace these flagged "bad password" entries with ``***REMOVED***``. It is important that you read through this file to make sure there are not exceptions that you want to remove and exclude with tartufo!
+   Use what you've learned to scan your repo!
 
    .. code-block:: console
 
-      # Create a "Bad password" file
-      BADSTRINGARR=`cat ${TARTUFOSCANPATH}/* | jq .strings_found | grep -v '\[' | grep -v '\]'`
-      readarray -t PEWPEWPEWARR <<<"$BADSTRINGARR"
-      # shows entires in array, may be larger than the earlier word count
-      echo ${#PEWPEWPEWARR[@]};
-      SORTED_UNIQUE_ENTRIES=($(echo "${PEWPEWPEWARR[@]}" | tr ' ' '\n' | sort -u | tr '\n' ' '))
-      echo ${#SORTED_UNIQUE_ENTRIES[@]};
-      printf "%s\n" "${SORTED_UNIQUE_ENTRIES[@]}" > badpwz.txt
-      # remove leading/trailing quotes and trailing commas
-      sed -i -e 's/\,$//g' -e 's/\"$//g' -e 's/^\"//g' badpwz.txt
-      
-   Now you have a "bad password" file! Take a look through it, see if anything is wrong, these values will be replaced in your code history
+      # Run Tartufo on your repo and create a list of high entropy items to remove:
+      poetry run tartufo --regex --json --cleanup ${GITHUBREPO} | jq -r '.found_issues[].matched_string' | sort -u > remove.txt
 
+   Now you have a "bad password" file! Take a look through it, see if anything is wrong. This file will be used by ``BFG`` and replace these flagged "bad password" entries with ``***REMOVED***``. It is important that you read through this file to make sure there are not exceptions that you want to remove and exclude with tartufo!
 
-#. Cleanup repo using ``BFG`` and the above passwords file
+
+#. Cleanup repo using ``BFG`` and the above remove.txt file
 
    There's a very slick tool designed to cleanup git commit history called ``BFG``: https://rtyley.github.io/bfg-repo-cleaner/. By default ``BFG`` doesn't modify the contents of your latest commit on your master (or 'HEAD') branch, even though it will clean all the commits before it. This of course means if you have active code with "bad passwords" tartufo will still fail, but let's take the bulk of the old entries out first.
 
    .. code-block:: console
 
-      # Cleanup with
+      # Cleanup with BFG
       wget https://repo1.maven.org/maven2/com/madgag/bfg/1.13.0/bfg-1.13.0.jar
-      git clone --mirror git@${GITHUBADDRESS}:${GITHUBPROJECT}/${GITHUBREPO}
       # Make a backup
       cp -r ${GITHUBREPO} backup_${GITHUBREPO}
-      java -jar bfg-1.13.0.jar --replace-text badpwz.txt ${GITHUBREPO}
+
+      do java -jar bfg-1.13.0.jar --replace-text remove.txt ${GITHUBREPO}
```

Comment:
Why `do` here?

Reply:
neglected to push up the most recent fix, good catch!